### DISCUSSION TITLE
Unable to flash inav version 6.1.1 on speedybee f405 v3

### DISCUSSION THREAD

**Discussion by serhii-bilyk-intellias** [iNavFlight/inav#9515] [Q&A] [answered]
Hi, after I flashed my speedybee f405 flight controller with inav 6.1.1. I am getting this error when I connect:
```
2023-11-26 @ 13:13:10 -- This firmware version is not supported. This version of Configurator supports firmware from 7.0.0 to 8.0.0 (excluded)
```
How can I fix it ?

**Comment by MrD-RC** [comment, marked as answer]
You are using the wrong Configurator. 

You are using Configurator 7.0 RC 1 or 2.

You need to use Configurator 6.1.
